ejabberd_bridge/tests.py

```diff
@@ -133,9 +133,9 @@ def test_auth_does_not_exist(self):
         """
         user_id = "user_that_does_not_exists"
         token = "token"
-        with self.assertRaises(exceptions.AuthenticationFailed) as cm:
+        with self.assertRaises(ValueError) as cm:
```

Comment:
@fichie23 ValueError should be catch in `auth` method, and return False to ejabberd